src/components/NewBookForm.js

```diff
@@ -17,23 +17,27 @@ const NewBookForm = () => {
       }}
       className="form"
     >
-      <input
-        placeholder="Book title"
-        className="input"
-        name="title"
-        type="text"
-        value={title}
-        onChange={(e) => setTitle(e.target.value)}
-      />
-      <input
-        placeholder="Author"
-        className="input"
-        name="author"
-        type="text"
-        value={author}
-        onChange={(e) => setAuthor(e.target.value)}
-      />
-      <button type="submit" className="form-button">Add Book</button>
+      <h2 className="form-title">ADD NEW BOOK</h2>
+      <div className="inputs">
+        <input
+          placeholder="Book title"
+          className="input"
+          name="title"
+          type="text"
+          value={title}
+          onChange={(e) => setTitle(e.target.value)}
+        />
+
+        <input
+          placeholder="Author"
+          className="input"
+          name="author"
+          type="text"
+          value={author}
+          onChange={(e) => setAuthor(e.target.value)}
+        />
+        <button type="submit" className="form-button">Add Book</button>
+      </div>
```

Comment:
- [ Optional ] I think it would be better if you validated the `title` and `author` fields so as to prevent a user from adding an empty book on the Books page. Currently, it appears a user can add an empty book to the list. Please see the screenshot below and rectify this.

## Screenshot
![Screenshot from 2023-05-26 21-16-17](https://github.com/Ashnaali3255/bookstore/assets/16392046/e0425b62-6630-40ad-921a-38d8d5bc3324)